pkg/cloudprovider/provider/hetzner/provider.go

```diff
@@ -386,6 +414,9 @@ func (s *hetznerServer) Addresses() []string {
 	for _, fips := range s.server.PublicNet.FloatingIPs {
 		addresses = append(addresses, fips.IP.String())
 	}
+	for _, privateNetwork := range s.server.PrivateNet {
+		addresses = append(addresses, privateNetwork.IP.String())
```

Comment:
Is there a correct order here, e.g. should private addresses go before or after other addresses?

Reply:
Nom, doesn't matter